docs/entities/OfferReview.md

```diff
@@ -0,0 +1,24 @@
+# OfferReview
```

Comment:
Again, it is preferable to use single words in main entities, otherwise you may imply a relation that can confuse people. This would a be a plain Review.